src/hb.hh

```diff
@@ -173,9 +173,8 @@ struct _hb_alignof
 #ifndef HB_INTERNAL
 # if !defined(HB_NO_VISIBILITY) && !defined(__MINGW32__) && !defined(__CYGWIN__) && !defined(_MSC_VER) && !defined(__SUNPRO_CC)
 #  define HB_INTERNAL __attribute__((__visibility__("hidden")))
-# elif defined(__MINGW32__)
-   /* We use -export-symbols on mingw32, since it does not support visibility
-    * attribute. */
+# elif defined(__MINGW32__) || defined (_MSC_VER)
+   /* We use -export-symbols on mingw32 and __declspec(dllexport), since they do not support visibility attributes. */
```

Comment:
I don't like this change.  Instead, add the code to define HB_INTERNAL to __declspec(dllexport) *here*, instead of putting it in the build file or something.